archivesunleashed/aut
# Converts WARC RDD into a GraphX object, performs PageRank and converts into GraphML object

**The title of this pull-request should be a brief description of what the pull-request fixes/improves/changes. Ideally 50 characters or less.**

This PR converts an RDD of WARC into GraphX object, performs PageRank (dynamic/ static) and creates a GraphML file.
* * *

**GitHub issue(s)**:

If you are responding to an issue, please mention their numbers below.

*  #203 

# What does this Pull Request do?

Creates a new file ExtractGraphXSLS.scala that is to be called from an external script. PageRank is implemented and added to GraphML file.


# How should this be tested?

Run the following script:
[TestScript.txt](https://github.com/archivesunleashed/aut/files/2013937/TestScript.txt)
Check the GraphML file created at the specified path in script.


# Additional Notes:

Example:
* This change requires updating of the documentation.

Thanks in advance for your help with the Archives Unleashed Toolkit!
